crates/trie/src/nodes/branch.rs

```diff
@@ -25,7 +25,7 @@ impl<'a> BranchNode<'a> {
         hash_mask: TrieMask,
     ) -> impl Iterator<Item = H256> + '_ {
         let mut index = self.stack.len() - state_mask.count_ones() as usize;
-        (0..16).filter_map(move |digit| {
+        CHILD_INDEX_RANGE.clone().filter_map(move |digit| {
```

Comment:
this clone is unnecessary because `CHILD_INDEX_RANGE` is a constant
```suggestion
        CHILD_INDEX_RANGE.filter_map(move |digit| {
```